app-modules/alert/src/Policies/AlertPolicy.php

```diff
@@ -10,50 +10,57 @@ class AlertPolicy
 {
     public function viewAny(User $user): Response
     {
-        return $user->can('alert.view-any')
-            ? Response::allow()
-            : Response::deny('You do not have permission to view alerts.');
+        return $user->canOrElse(
+            abilities: 'alert.view-any',
+            denyResponse: 'You do not have permission to view alerts.'
+        );
     }
 
     public function view(User $user, Alert $alerts): Response
```

Comment:
Small nit and I know you didn't alter it here, but `$alerts` should be singular `$alert` throughout the class.